sorc/fbwndgfs.fd/makefile.GENERIC

```diff
@@ -1,81 +0,0 @@
-SHELL=/bin/sh
```

Comment:
I updated this in the port to WCOSS2. I either need this to stay for that port in `develop` or know how this is building in `develop` currently so I know where to add WCOSS2 support in the very near future.

Reply:
We are building this utility w/ CMake.  So any change in this file is moot in `develop`.